src/core/MREntity.js

```diff
@@ -65,6 +65,8 @@ export class MREntity extends MRElement {
         this.touch = false;
         this.grabbed = false;
         this.focus = false;
+
+        this.ignoreStencil = true
```

Comment:
question - this should make all entities ignored by masking, even div entities - meaning we're not masking at all anymore - is this needed in this pr?

if so - make sure to add `this.ignoreStencil = false` to all mrdiventity items (or at a basis, mrdiventity)

Reply:
ah it should be disabled for MREntity but enabled for MRDivEntity, I'll fix that. This should achieve the desired effect of masking UI elements but allowing 3D content to be clipped in an mr-volume.

Reply:
@hanbollar looks like this is already the case in MRDivEntity, which sets it's values after the MREntities constructor runs.

https://github.com/Volumetrics-io/mrjs/blob/c802e57ef5f70f8f0ee3f25f915844c0550e745f/src/core/MRDivEntity.js#L44

Reply:
ah shoot - nah you're all good there, just checked and MRDivEntity already has the this.ignoreStencil = false ~, good to merge